lnrpc/rpc.proto

```diff
@@ -2069,6 +2069,7 @@ message LightningNode {
     string alias = 3 [ json_name = "alias" ];
     repeated NodeAddress addresses = 4 [ json_name = "addresses" ];
     string color = 5 [ json_name = "color" ];
+    repeated Feature features = 6 [ json_name = "features" ];
```

Comment:
nit: we have `[json_name` (without a space) more than `[ json_name` (with space) in `rpc.proto`, not sure of which is correct, just noticed it.

Reply:
yeah lol both styles are used throughout the file, i just tried to match the surrounding definitions